package.json

```diff
@@ -60,7 +60,9 @@
     "history": "^4.6.3"
   },
   "dependencies": {
+    "history": "^4.7.2",
```

Comment:
`history` should be listed in `peerDependencies` only.

Reply:
woops installed it to test but forgot about this thanks